src/_pytest/config/argparsing.py

```diff
@@ -61,7 +76,7 @@ def getgroup(self, name, description="", after=None):
         self._groups.insert(i + 1, group)
         return group
 
-    def addoption(self, *opts, **attrs):
+    def addoption(self, *opts: str, **attrs: Any) -> None:
```

Comment:
Yeah, TypeScript has this feature, but mypy doesn't AFAIK. The other option is to duplicate, but I wouldn't want to do that.

Reply:
Could a typevar be used here, to be used in those (3+) places?
But this could also be done later anyway - I've just thought it would be really helpful with completion of those functions.
(You have probably noticed also that documentation in this regard is outdated etc, and it's messy in general, so that would help there.)

Reply:
> Could a typevar be used here, to be used in those (3+) places?

Not that I know of...